perlmod/Fink/PkgVersion.pm

```diff
@@ -5218,6 +5221,7 @@ sub run_script {
 	my $phase = shift;
 	my $no_expand = shift || 0;
 	my $nonroot_okay = shift || 0;
+	my $no_sandbox_okay = shift || 0;
```

Comment:
I.e. the prior sixth parameter in the subroutine call was $ignore_result, and by _inserting_ $no_sandbox_okay at the sixth position rather than _appending_ it on the seventh, this changes the meaning of the sixth parameter.